it-and-security/lib/configuration-profiles/macos-ensure-show-status-bar-is-enabled.mobileconfig

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>PayloadContent</key>
+		<array>
+			<dict>
+				<key>PayloadDisplayName</key>
+				<string>Ensure Show Status Bar Is Enabled</string>
+				<key>PayloadType</key>
+				<string>com.apple.Safari</string>
+				<key>PayloadIdentifier</key>
+				<string>com.fleetdm.cis-Ensure Show Status Bar Is Enabled</string>
+				<key>PayloadUUID</key>
+				<string>708B39DB-E2B7-405C-A523-88F3DDC8DFFC</string>
+				<key>ShowOverlayStatusBar</key>
+				<true/>
+			</dict>
+		</array>
+		<key>PayloadDescription</key>
+		<string>Ensure Show Status Bar Is Enabled</string>
+		<key>PayloadDisplayName</key>
+		<string>Ensure Show Status Bar Is Enabled</string>
+		<key>PayloadIdentifier</key>
+		<string>com.fleetdm.cis-Ensure Show Status Bar Is Enabled</string>
```

Comment:
Same here and use a different identifier than the one above (check patters we use from other configuration profiles on this directory).